tests/components/unifi/test_device_tracker.py

```diff
@@ -274,12 +275,11 @@ async def test_tracked_wireless_clients_event_source(
     assert hass.states.get("device_tracker.client").state == STATE_HOME
 
     # Change time to mark client as away
-    new_time = dt_util.utcnow() + controller.option_detection_time
-    with patch("homeassistant.util.dt.utcnow", return_value=new_time):
-        async_fire_time_changed(hass, new_time)
-        await hass.async_block_till_done()
+    freezer.tick(controller.option_detection_time + timedelta(seconds=1))
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done()
 
-    assert hass.states.get("device_tracker.client").state == STATE_HOME
+    assert hass.states.get("device_tracker.client").state == STATE_NOT_HOME
 
```

Comment:
The comment is right but the test was wrong because it didn't move time far enough to actually go away/not home